monsteer/CMakeLists.txt

```diff
@@ -18,5 +18,5 @@ set(MONSTEER_LINK_LIBRARIES PUBLIC Brion Lunchbox zeq)
 
 common_library(Monsteer)
 
+add_subdirectory(brionplugin)
```

Comment:
Why not simply "plugin"?

You can also reverse the directories ($ROOT/brion/monsteer) and create the namespace brion::monsteer. This is short, clear and can be consistent.